src/components/breadcrumb-bar/__fixtures__/terraform-plugin-sdkv2-nav-data.json

```diff
@@ -0,0 +1,187 @@
+[
```

Comment:
This nav-data fixture is a chonky part of the PR. It felt necessary to build a meaningful test case to cover new functionality in `get-docs-breadcrumbs`.